packages/runtime/container-runtime/src/test/gc/gcHelpers.spec.ts

```diff
@@ -86,22 +86,22 @@ describe("Garbage Collection Helpers Tests", () => {
 				expectedShouldAllowValue: false,
 			},
 			{
-				persisted: { sweepGeneration: 0 },
+				persisted: { sweepGeneration: 0, tombstoneGeneration: 0 },
```

Comment:
We no longer advise that any host passes in only `gcSweepGeneration` so the persisted state should never by only `sweepGeneration`